src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+const CHECK_STATUS = './library/categories/CHECK_STATUS';
+
+const stateInit = {
+  type: 'action',
+  checked: true,
+  payload: 'Under construction',
+};
```

Comment:
- [ ] Kindly implement this requirement asked by the previous reviewer

Reply:
Please can you Kindly explain further what am to do

Reply:
Your initial state should be an empty array here

Reply:
thanks @GzimAsani  for the information changes made 👍